code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -107,11 +107,11 @@ var/veil_thickness = CULT_PROLOGUE
 				to_chat(M,"<span class='warning'>Be mindful, overzealous conversions and soul trapping will bring us unwanted attention. You should focus on the objective with your current force.</span>")
 
 
-//CULT_PROLOGUE		Default thickness, only communication and raise structure runes enabled
+//CULT_PROLOGUE		Default thickness, only communication and raise structure runes enabled.
 //CULT_ACT_I		Altar raised. cultists can now convert.
-//CULT_ACT_II		Cultist amount reached. cultists are now looking for the sacrifice
-//CULT_ACT_III		Sacrifice complete. cult is now going loud, spreading blood and protecting bloodstones while the crew tries to destroy them
-//CULT_ACT_IV		Bloodspill threshold reached. bloodstones become indestructible, rift opens above one of them. cultists must open it, crew must close it.
```

Comment:
??

I mean both lines are technically correct, not sure why you'd change that though.